### PR TITLE
redo 142

### PR DESCRIPTION
**Pull request by timkpaine**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by timkpaine** [comment]
@conda-forge-admin, please rerender

**Comment by hmaarrfk** [comment]
Thank you for doing this. I realize it is arbitrary, it is just hard to follow all active users on conda-forge. 

Therefore, this does show me (or an other core member that hasn't interacted with you) that you are committed and have the ability required to maintain this particular feedstock.
